assets/scss/base/_buttons.scss

```diff
@@ -32,6 +33,24 @@ button, .button {
   @include core-19();
 }
 
+.button--blue {
+  @extend .button--get-started;
```

Comment:
This shouldn't extend this class, both classes should be applied to the markup...

`class="button button--get-started button--blue"`